VnE/Source/Extensions/ChemPack/ui/ui_instruments.py

```
@@ -45,6 +45,17 @@ def setupUi(self, Dialog):
 
         self.horizontalLayout_2.addWidget(self.pushButton_2)
 
+        self.pushButton_3 = QPushButton(Dialog)
+        self.pushButton_3.setObjectName(u"pushButton_3")
+
+        self.horizontalLayout_2.addWidget(self.pushButton_3)
+
+        self.pushButton_4 = QPushButton(Dialog)
+        self.pushButton_4.setObjectName(u"pushButton_2")
```

Ruff (0.14.14)

[warning] Remove unicode literals from strings

Remove unicode prefix

(UP025)